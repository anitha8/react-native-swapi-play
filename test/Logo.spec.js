import 'react-native';
import React from 'react';
import Logo from '../src/components/Logo';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <Logo />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});