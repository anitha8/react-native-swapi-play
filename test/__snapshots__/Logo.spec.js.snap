// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 80,
        "width": 80,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "white",
        "fontWeight": "bold",
        "marginTop": 20,
      }
    }
  >
    SWAPI LOGIN
  </Text>
</View>
`;
